src/server/api/schema.js

```diff
@@ -664,6 +664,13 @@ const rootMutations = {
 
       const { contactNumber, text } = message
 
+      if (text.length > (process.env.MAX_MESSAGE_LENGTH || 280)) {
```

Comment:
For backwards compatibility, please test only IF there is a process.env.MAX_MESSAGE_LIMIT, i.e.
`if (process.env.MAX_MESSAGE_LIMIT && text.length > process.env.MAX_MESSAGE_LENGTH) {`